cranelift/codegen/src/isa/arm64/inst/args.rs

```diff
@@ -137,7 +137,7 @@ pub enum MemLabel {
     /// ConstantPoolRel during emission. The isel should use it by emitting a
     /// Load64(ExtName(..)) to get the *address* of the external symbol, then
     /// calling/loading/storing that address as appropriate.
-    ExtName(ExternalName, i64),
+    ExtName(ExternalName, i64, ir::SourceLoc),
```

Comment:
I reworked `MemLabel` a bit today, so this will need a little adaptation (sorry!) -- there's now a dedicated `LoadExtName` instruction that loads the value of an external symbol.